include/z64save.h

```diff
@@ -10,6 +10,13 @@ struct GameState;
 struct PlayState;
 struct FileSelectState;
 
+typedef enum {
+    /* 0 */ FS_AUDIO_STEREO,
+    /* 1 */ FS_AUDIO_MONO,
+    /* 2 */ FS_AUDIO_HEADSET,
+    /* 3 */ FS_AUDIO_SURROUND
+} AudioOption;
```

Comment:
It's annoying because there's two distinct enums for the same thing, one in save context and one in audio context. This name just came from OoT but could always change this prefix to `SAVE_`

Reply:
Well if it's referring to something in SaveContext probably it should use `SAVE_`, then (maybe there is something in file_choose itself that uses it but surely this is the primary context for it).